homeassistant/components/wled/light.py

```diff
@@ -48,6 +60,12 @@
     """Set up WLED light based on a config entry."""
     coordinator: WLEDDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
 
+    platform = entity_platform.current_platform.get()
+
+    platform.async_register_entity_service(
+        SERVICE_EFFECT, SERVICE_EFFECT_SCHEMA, "async_effect",
```

Comment:
Feel free to inline these constants. Not much added benefit adding them to the top.

Reply:
inlined.